c_src/brotli_nif.c

```diff
@@ -73,6 +74,39 @@ brotli_encode(ErlNifEnv *env, int argc, const ERL_NIF_TERM argv[])
     return enif_make_binary(env, &encoded);
 }
 
+static ERL_NIF_TERM
+brotli_decode(ErlNifEnv *env, int argc, const ERL_NIF_TERM argv[])
+{
+  ErlNifBinary data, decoded;
+
+  if (argc != 1) {
+    return (BADARG);
+  }
+
+  if (!enif_inspect_iolist_as_binary(env, argv[0], &data)) {
+    return (BADARG);
+  }
+
+  size_t length = data.size;
+  size_t output_length = data.size * 100; /* FIXME: how to calculate optimal value??? */
```

Comment:
I think we'd better to use stream api instead of allocating large memory block upfront.